gatsby-node.esm.js

```diff
@@ -378,6 +378,7 @@ exports.createPages = async ({actions, graphql}) => {
           name
           status
           a11yReviewed
+          importPath
```

Comment:
We only need to add `importPath` here if we're using it to build pages, which it appears we're not. We should be able to remove this line and the other one in this file 👍 Not a blocker tho!

Reply:
Hmm, okay. Thanks! I'll remove it now.